src/redux/car_list/carListSlice.js

```diff
@@ -0,0 +1,99 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const FETCH_CAR_LIST = 'cars/fetchCarList';
+const UPDATE_FETCH_CAR_LIST = 'cars/updateFetchCarList';
+
+const initialState = {
+  status: 'default',
+  cars: [],
+  response: '',
+  indexes: [],
+  delay: true,
+};
+
+const optionalBody = (body) => {
+  if (body === null) {
+    return null;
+  }
+  return JSON.stringify(body);
+};
+
+const fetchAPI = async (method, endPoint, body = null) => {
+  const request = await fetch(`http://localhost:4000/api/v1/${endPoint}`, {
```

Comment:
Maybe we can create an env variable in one file to get the API URL and use it all API calls, in that way when we publish this is only needed to change in one place the URL, for [example ](https://stackoverflow.com/questions/49579028/adding-an-env-file-to-react-project)